repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/splitheaders/addHeader/add-header-request.cfg.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<add-headers xmlns="http://docs.openrepose.org/repose/add-header/v1.0">
+    <request>
+        <header name="request-test-1">split</header>
```

Comment:
why are there values in these elements?

Reply:
ohh, i see this isn't config for the header splitting filter. man the config looks almost the same.